### PR TITLE
Applied new design, updated styles and layout

### PR DESCRIPTION
**Pull request by didiepetrova**

### CODE REVIEW

File: sample-file-open-pdf/index.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
     </div>
     <div class="subfooter">
         <div class="row">
-            <div class="col u-color-dark" >
+            <div class="col u-color-accent" >
                 <span>Note: For Android and WP8 you need a PDF reader installed.</span>
```
Comment (TsvetinaIvanova) [inline sample-file-open-pdf/index.html]: You can change WP8 to either WP or Windows Phone. It is better not to state the OS version, in my opinion.

You can consider making the screenshot on iPhone 5 in the simulator to preserve the original size.

**Comment by TsvetinaIvanova** [comment]
You can consider making the screenshot on iPhone 5 in the simulator to preserve the original size.

**Comment by TsvetinaIvanova** [comment]
:+1:

File: sample-file-open-pdf/.abproject [hunk 2]
```diff
@@ -1,72 +1,55 @@
...
+  "AndroidHardwareAcceleration": "true",
+  "BundleVersion": "1.0",
+  "AndroidVersionCode": "1",
+  "iOSDeviceFamily": [
+    "1",
+    "2"
+  ],
+  "FrameworkVersion": "4.0.0",
+  "CorePlugins": [
+    "cordova-plugin-device",
```
Comment (Mike4o) [inline sample-file-open-pdf/.abproject]: Do we need the device plugin in this sample?

**Comment by Mike4o** [comment]
You can also take the screenshot in the AppBuilder Simulator for iPhone 6 to keep the correct dimensions - 640x1136

**Comment by TsvetinaIvanova** [comment]
It's iPhone 5 in the simulator with this dimensions, not 6.

**Comment by didiepetrova** [comment]
The Device Plugin is mandatory! :)

**Comment by Mike4o** [comment]
:+1:

**Pull request merged**